apis/expansion/v1alpha1/expansiontemplate_types.go

```diff
@@ -58,7 +58,6 @@ type GeneratedGVK struct {
 // +kubebuilder:resource:path="expansiontemplate"
 // +kubebuilder:resource:scope="Cluster"
 // +kubebuilder:subresource:status
-// +kubebuilder:storageversion
```

Comment:
v1alpha1 should remain the storage version for at least one release to allow for rollbacks.

Reply:
Done